### PR TITLE
Access Investigation - ID fix

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Description
ID was changed due to SDK format while deprecating the old playbook here https://github.com/demisto/content/pull/7315/files. 
This PR changes it back to the old ID (though it's different than the name, which should fail this build).

## Minimum version of Demisto
4.5.0

## Does it break backward compatibility?
No

### CODE REVIEW

**Comment by Itay4** [comment]
hold on with this

**Comment by Itay4** [comment]
@idovandijk `Packs/AccessInvestigation/ReleaseNotes/1_1_1.md: [RN107] - No release note entry was found for the playbook "Access Investigation - Generic" in the AccessInvestigation pack. Please rerun the update-release-notes command without -u to generate an updated template.`

File: Packs/AccessInvestigation/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1 @@
+<!-- comment -->
```
Comment (amshamah419) [inline Packs/AccessInvestigation/ReleaseNotes/1_1_1.md:1]:
```suggestion
<!-- 
#### Playbooks
##### Access Investigation - Generic
- No need for RNs
 -->
```

**Comment by Itay4** [comment]
build fails on approved validation
merging

**Pull request merged**